.travis.yml

```diff
@@ -21,6 +24,7 @@ deploy:
   on:
     tags: true
     python: 3.7  # prevent deploying multiple times (an therefore causing deploys to fail)
+    condition: -z $TOXENV
```

Comment:
can you add a comment on what problem this lines solves?

Reply:
Sure, done.

Docs on `condition`: https://docs.travis-ci.com/user/deployment#conditional-releases-with-on

Reply:
I'm not able to test it, but the intent is for the deployment to kick in when running the "normal" 3.7 build (I assume $TOXENV is empty there), not (also) the typing one.

Reply:
LGTM